Automattic/studio
# Use PHP constants from `wp-now`

Related to https://github.com/Automattic/studio/pull/225.

## Proposed Changes

- Instead of defining new constants for the available and default PHP versions, use the values provided by `wp-now` and Playground.

<img width="900" alt="Screenshot 2024-06-11 at 15 34 02" src="https://github.com/Automattic/studio/assets/14905380/44fe87b9-4e96-44c3-b74f-9fc2ab997112">

## Testing Instructions



1. Create a new site.
2. New site should launch with a default version of PHP 8.0.
3. Navigate to the Studio settings for the site.
4. Edit the PHP version to a different PHP version.
5. Observe that the PHP versions listed are the same as the ones offered in https://playground.wordpress.net/.
6. Verify the server restarts and the site is running the new PHP version.

## Pre-merge Checklist
